index.html

```diff
@@ -4,9 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Day 1: Solo Build</title>
+    <link rel="stylesheet" href="./src/styles.css">
 </head>
 <body>
-    
+    <H1>Hi I'm Eli, I like <a href="#pets">my pets</a> and <a href="#kayak">kayaking!</a> Below are my pets and a couple fun  words about kayaking.</H1>
+    <h2 id="pets">My Pets are and look like:</h2>
+    <ul>
+        <li><a href="#smoke">Smokey Joe |</a></li>
+        <li><a href="#rory">Rory |</a></li>
+        <li><a href="#tyb">Tybalt |</a></li>
+        <li><a href="#ranger">Ranger</a></li>
+    </ul>
+    <h3 id="smoke">Smokey Joe:</h3>
+    <p>Obsessing over baby chicks!"cuckadoodledoo!"</p>
+    <img id="myPets" src="./Assests/Smokey Joe.jpg" alt="Picture of Pyrenese mixbreed dog">
+    <h3 id="rory">Rory:</h3>
+    <p>Seen here clearly attempting to wake someone up too early</p>
+    <img id="myPets" src="./Assests/Rory.jpg" alt="Picture of yellow Lab/Aussie mixbreed dog">
+    <h3 id="tyb">Tybalt:</h3>
+    <p>Clearly annoyed someone took a photo of him for this page</p>
+    <img id="myPets" src="./Assests/Tybalt.jpg" alt="Picture of Siamese mixbreed cat">
+    <h3 id="ranger">Ranger:</h3>
+    <p>Perched in preparation for an attack on the unknown</p>
+    <img id="myPets" src="./Assests/Ranger.jpg" alt="Picture of Bengal mixbreed cat">
+    <hr>
+    <h2 id="kayak">Fun kayaking jargon no one understands</h2>
+    <ol>
+        <li><a href="#boof">Boof!</a></li>
+        <li><a href="#steeze">Steeze</a></li>
+        <li><a href="#gnar">Gnar</a></li>
+    </ol>
+    <h3 id="boof">Boof</h3>
+    <p>Boof is a verb meaning: to paddle at a location and intend to launch off of it, most often a rock with some water flowing over at least part of it. Used in a sentence: ' Bruh, follow me and <strong>boof</strong> this ledge.'</p>
+    <img src="./Assests/Boof.jpg" alt="">
+    <h3 id="steeze">Steeze</h3>
+    <p>The word steeze is most often used in a context as to refer to a move someone made as really impressive or cool. Used in a sentence 'Dude, that boof was so <strong>steeze</strong>!'</p>
+    <img src="./Assests/Steeze.jpg" alt="">
+    <h3 id="gnar">Gnar</h3>
+    <p>the word Gnar refers to rapids that are really big, hard, impossible to kayak. Used in a sentence: 'that rapid is <strong>gnar</strong>!'</p>
+    <img src="./Assests/Gnar.jpg" alt="">
 </body>
```

Comment:
a `<footer></footer>` tag with a link back to your github is a great touch - that way all your fans can find you! :-)